packages/react-components/react-tags/library/src/components/InteractionTag/useInteractionTag.tsx

```diff
@@ -16,15 +16,20 @@ export const useInteractionTag_unstable = (
   props: InteractionTagProps,
   ref: React.Ref<HTMLDivElement>,
 ): InteractionTagState => {
-  const { handleTagDismiss, size: contextSize, appearance: contextAppearance } = useTagGroupContext_unstable();
+  const {
+    handleTagDismiss,
+    size: contextSize,
+    disabled: contextDisabled,
+    appearance: contextAppearance,
+  } = useTagGroupContext_unstable();
 
   const id = useId('fui-InteractionTag-', props.id);
 
   const interactionTagPrimaryId = useId('fui-InteractionTagPrimary-');
 
   const {
     appearance = contextAppearance ?? 'filled',
-    disabled = false,
+    disabled = contextDisabled ?? false,
```

Comment:
I think it would make more sense to force tag/interactionTag to be disabled always when tagGroup is disabled. 
With the current implementation `<TagGroup disabled><Tag disabled={false} /></TagGroup>` is still possible, but it doesn't make sense to have this combination.

Reply:
Also it would be nice to have a unit test for this :)